tests/subsys/ipc/pbuf/prj.conf

```diff
@@ -1,6 +1,7 @@
 CONFIG_ZTEST=y
 CONFIG_ZTRESS=y
 CONFIG_ZTEST_SHUFFLE=y
+CONFIG_ZTEST_REPEAT=y
```

Comment:
Why this Kconfig option enabled here without any of the ones defining the repeat count?

Reply:
Previously this prj.conf enabled only CONFIG_ZTEST_SHUFFLE, it means that it left the CONFIG_ZTEST_SHUFFLE_* with the default value (3). So, I've left the same approach. Enabling CONFIG_ZTEST_REPEAT, leaving the CONFIG_ZTEST_*_REPEAT_COUT at the default value (3).

Reply:
Ah, I see. There is still one occurence of this, in `tests/net/lib/lwm2m/block_transfer/prj.conf`.

Reply:
True, I had to miss it out.